sdks/python/apache_beam/typehints/typehints_test.py

```diff
@@ -918,139 +776,16 @@ def test_must_be_single_return_type(self):
       def unused_foo():
         return 4, 'f'
 
-  def test_type_check_violation(self):
-    @check_type_hints
-    @with_output_types(int)
-    def foo(a):
-      return 'test'
-    with self.assertRaises(TypeCheckError) as e:
-      m = 4
-      foo(m)
-
-    self.assertEqual("Type-hint for return type violated. Expected an "
-                     "instance of {}, instead found an instance of "
-                     "{}.".format(int, type('test')),
-                     e.exception.args[0])
-
-  def test_type_check_simple_type(self):
-    @with_output_types(str)
-    def upper(a):
-      return a.upper()
-    self.assertEqual('TEST', upper('test'))
-
   def test_type_check_composite_type(self):
     @with_output_types(typehints.List[typehints.Tuple[int, int]])
-    def bar():
+    def unused_bar():
       return [(i, i+1) for i in range(5)]
 
-    self.assertEqual([(0, 1), (1, 2), (2, 3), (3, 4), (4, 5)], bar())
-
   def test_any_return_type_hint(self):
     @with_output_types(typehints.Any)
-    def bar():
+    def unused_bar():
       return 'foo'
 
-    self.assertEqual('foo', bar())
-
-
-class CombinedReturnsAndTakesTestCase(TypeHintTestCase):
```

Comment:
Could you explain why these were all deleted?

Reply:
I removed the tests that used `@check_type_hints`, since that decorator doesn't exercise the actual type checking code paths. `PTransformTypeCheckTestCase` has coverage for `runtime_type_check = True`.